### ISSUE TITLE
Unmodified NB is being constantly checkpointed

### ISSUE THREAD

**Issue by sam-s** [millejoh/emacs-ipython-notebook#131]
I get these messages in `*Message*` every few seconds:

```
ein: [info] Writing notebook ZZZ.ipynb as nbformat 4.
ein: [info] Saving content successful with status nil
ein: [info] Notebook is saved.
ein: [info] Checkpoint checkpoint for ZZZ.ipynb generated.
```

I also see 

```
[I 14:16:16.644 NotebookApp] Saving file at /ZZZ.ipynb
[W 14:16:16.652 NotebookApp] Saving untrusted notebook ZZZ.ipynb
```

in the terminal every minute.
I think this is far far too often.
how do I control this?

**Comment by sam-s** [comment]
found `ein:notebook-autosave-frequency` -- still wrong, the NB should be autosaved only when it has been modified

**Comment by millejoh** [comment]
I just pushed some changes that make ein much less talkative when it saves notebooks, so that is half the solution. I'll add in some checks to ensure the notebook is only saved when it has been modified.

**Comment by millejoh** [comment]
Now should only autosave if a worksheet buffer has been modified. Let me know how that works for you.

**Comment by xelibrion** [comment]
looks like it's been fixed in 1c04701a90ca9f48a5eefed529992ad98eb491e4
